### DISCUSSION TITLE
Remember settings for navigraph page inside EFB

### DISCUSSION THREAD

**Discussion by felixfong227** [flybywiresim/aircraft#7211] [Help] [answered]
Branch: stable + development

stable version: v0.8.1
development: 1902cbf

---

first of all, since I haven't played MSFS for a while, seeing a lot of great changes to this project, and has added tons of new and cool features ever since just put a big smile on my face, thank you to all of the devs and the community 😄 

Currently, when navigating away from the Navigraph page inside EFB to other pages, all of the states from the previous visit will be wiped gone, so it's a added step to re-type the airport code and the correct chart again, the zoom level of the PDF is also incorrect

Am I doing something wrong or have I just misconfigured the EFB or it's just a bug?

**Comment by HowlingWolf136** [comment]
Same with me :-(

**Reply by ErickSharp** [reply]
Please update to latest development version. This has been addressed recently in FlyPadOS 3.

**Comment by felixfong227** [comment, marked as answer]
Closing this thread, FlyPadOS3 solved this issue wonderfully
